Dockerfile

```
@@ -1,9 +1,12 @@
 FROM messense/rust-musl-cross:x86_64-musl as builder
 
+WORKDIR /
+
 RUN cargo new dotenv-linter
 COPY Cargo.toml ./
-COPY src ./src
-COPY benches ./benches
+COPY src src
+COPY benches benches
+COPY .cargo .cargo
```

```suggestion
ENV CARGO_NET_GIT_FETCH_WITH_CLI=true
```
Maybe we will just use the environment variable instead of adding a file?

---

I believe it's more extendable way, we probably would need to add other configs in the future